README_de-DE.md

```diff
@@ -0,0 +1,432 @@
+If applied, this commit will: <commit message>
+```
+
+Beispiel:
+
+```
+# Good
+If applied, this commit will use InventoryBackendPool to retrieve inventory backend
+```
+
+```
+# Bad
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+```
+
+### Große Anfangsbuchstaben
+
+```
+# Good
+Add `use` method to Credit model
+```
+
+```
+# Bad
+add `use` method to Credit model
+```
+
+Diese Regelung macht grammatikalisch schlichtweg Sinn, da man den grammatikalischen Regeln bzgl. Großbuchstaben am Satzanfang gerecht wird.
+
+Diese Regelung kann von Entwickler zu Entwickler, Team zu Team oder auch Sprache zu Sprache unterschiedlich sein.
+
+Großschreiben oder nicht, wichtig ist nur, dass man bei einer Variante bleibt.
+
+### Versuche _genau_ zu beschreiben, was die Änderung macht, auch ohne dass man in den Code gucken muss.
+
+```
+# Good
+Add `use` method to Credit model
+
+```
+
+```
+# Bad
+Add `use` method
+```
+
+```
+# Good
+Increase left padding between textbox and layout frame
+```
+
+```
+# Bad
+Adjust css
+```
+
+Dies ist in vielen Situationen (z.B.: mehrer kleine Commits, Refactoring) sinnvoll um besser zu verstehen, was sich der Entwickler gedacht hat.
+
+### "Warum", "Wozu", "Wie" und andere Details gehören in die Beschreibung der Commit-Message
+
+```
+# Good
+Fix method name of InventoryBackend child classes
+
+Classes derived from InventoryBackend were not
+respecting the base class interface.
+
+It worked because cart was calling the backend implementation
+incorrectly.
+```
+
+```
+# Good
+Serialize and deserialize credits to json in Cart
+
+Convert the Credit instances to dict for two main reasons:
+
+  - Pickle relies on file path for classes and we do not want to break up
+    everything if a refactor is needed
+  - Dict and built-in types are picklable by default
+```
+
+```
+# Good
+Add `use` method to Credit model
+
+Change from namedtuple to class because we need to
+setup a new attribute (in_use_amount) with a new value
+```
+
+Titel bzw. Betreff der Commit-Message und die Beschreibung müssen mit einer Leerzeile getrennt werden. Alle weiteren Leerzeilen oder Absätze werden als Teil der Beschreibung gewertet.
+
+Trenn- oder Aufzählungszeichen, wie z.B.: `-`, `*` oder `\` verbessern die Lesbarkeit der Beschreibung.
+
+### Vermeide allgemeingültige oder kontextfreie Beschreibungen
+
+```
+# Bad
+Fix this
+
+Fix stuff
+
+It should work now
+
+Change stuff
+
+Adjust css
+```
+
+### Limitiere die Länge Commit-Message
+
+[Es wird empfohlen (Quelle in ENG)](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines) maximal 50 Zeichen für den Titel/Betreff und 72 Zeichen für die Nachricht zu verwenden.
+
+### Bleibe in einer Sprache
+
+Dies gilt vor allem für Entwickler für die Englisch nicht die Muttersprache ist. Logischerweise sollte man sich immer dem etablierten Standard anpassen – ganz besonders bei Repositories mit Entwicklern aus vielen verschiedenen Ländern.
```

Comment:
This is not exactly translating the original one, I'd try to keep both versions the same.

Reply:
I see what you mean, although the version I forked already differs from the current HEAD. Feel free to suggest a translation and I'm happy to update mine.

Reply:
I see, you were using the version before this commit https://github.com/RomuloOliveira/commit-messages-guide/commit/8c52dad1ad5b114e809a4f538203b57dc0777923

ok, then I would say it is fine. It is the problem of supporting several languages, when one is updated, the others need to be updated again. I think there are still some things to improve in the English version. So we can leave the German translation as it is, an in some time, synchronise it with the English one again. :wink:

Reply:
Yes, I agree on that. ;)